src/components/App.jsx

```diff
@@ -1,10 +1,14 @@
 import React from "react";
 import ContactsApp from "./ContactsApp";
+import ContactsHeader from "./ContactsList/ContactsHeader";
 
 const App = () => (
   <div className="app-wrapper c-layout">
     <main className="app-content">
-      <ContactsApp />
+      <ContactsHeader />
+      <div role="contentinfo">
+        <ContactsApp />
+      </div>
```

Comment:
I don't think so.
What is the purpose to move the `ContactsHeader` outside the `ContactsApp`?

Si j'ai mis un composant `ContactsApp` c'est pour permettre de controler la `ContactsList` avec les filtres de la `ContactsHeader`. Là, c'est rendu impossible et je ne comprends pas le besoin. Peux-tu m'expliquer ?

Reply:
OK alors à ce moment faut mettre `ContactApp` plus haut et renommer des choses mais j'ai besoin d'un DOM spécifique pour obtenir le layout souhaiter en UX.